src/PKSim.Core/Snapshots/OriginData.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using OSPSuite.Utility.Collections;
+
+namespace PKSim.Core.Snapshots
+{
+   /// <summary>
+   ///    Should only be used from matlab or batch component to create individual based on user data.
+   ///    This structure will then be converted into a Core OriginData
+   /// </summary>
+   public class OriginData
+   {
+      private readonly Dictionary<string, string> _cmCache;
+
+      /// <summary>
+      ///    Id of species (as defined in the database)
+      /// </summary>
+      [Required]
+      public string Species { get; set; }
+
+      /// <summary>
+      ///    Id of population (as defined in the database)
+      /// </summary>
+      public string Population { get; set; }
+
+      /// <summary>
+      ///    Id of gender
+      /// </summary>
+      public string Gender { get; set; }
+
+      /// <summary>
+      ///    Age of individual to create in [year]. This is a mandatory input for age dependent species. If not defined, set to NaN
+      /// </summary>
+      public Parameter Age { get; set; }
+
+      /// <summary>
+      ///    Gestational age of individual to create in [weeks]. If not defined, set to 40
+      /// </summary>
+      public Parameter GestationalAge { get; set; }
```

Comment:
is is possible to use such a class in Matlab? Or does it have to be only primitive type (double, string etc..)